### ISSUE TITLE
Setting maximum-scale in viewer.html is an accessibility issue

### ISSUE THREAD

**Issue by cahrens** [mozilla/pdf.js#7755]
In the various versions of viewer.html (for example, https://github.com/mozilla/pdf.js/blob/69066234626775955afdffdc48544794b9817fb5/web/viewer.html#L26), _maximum-scale=1_ is set on the _meta name="viewport"_ element. This is an accessibility violation.

> Severity: critical
> Rule ID: meta-viewport
> Help URL: https://dequeuniversity.com/rules/axe/1.1/meta-viewport
> Message: <meta> tag disables zooming
> Html: <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
> Target: u'body > meta:nth-of-type(2)'

Steps to reproduce the problem:
1. Run axe rule set on web page with pdfjs viewer
2. See violation

As far as I can tell, there is no harm in simply removing _maximum-scale=1_.

**Comment by Snuffleupagus** [comment]
That line was added in PR #1743, and as mentioned in https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Viewport_width_and_screen_width:
> Mobile browsers handle orientation changes slightly differently. For example, Mobile Safari often just zooms the page when changing from portrait to landscape, instead of laying out the page as it would if originally loaded in landscape. If web developers want their scale settings to remain consistent when switching orientations on the iPhone, they must add a maximum-scale value to prevent this zooming, which has the sometimes-unwanted side effect of preventing users from zooming in:

Based on the above, it seems to me that `maximum-scale=1` was added deliberately, and removing it may have unintended side effects. (Note that the UI itself of the standard viewer isn't really intended to zoom, and there are open issues, e.g. #2582, tracking better document zooming on mobile.)

Hence it seems to me that WONTFIX may be the correct resolution for this issue.

**Comment by timvandermeij** [comment]
I agree with the above. It is necessary for proper display on mobile devices and in other situations where responsiveness is required.

**Comment by yurydelendik** [comment]
> This is an accessibility violation.

The viewer provides capability to zoom in/out. It allows better zooming capability than enabling viewport's one would allow. The only limitation is not allowing a pinch zoom, but that is covered by other bug.
